server/index.js

```diff
@@ -13,13 +13,53 @@ app.use(function(req, res, next) {
   next();
 });
 //API Service
-app.get('/api/volumes/symbols/:id', function(req, res){
-  PriceVolume.find({id: req.params.id}, (err, data)=>{
-    if(err) console.log(err);
-    res.setHeader('Content-Type', 'application/json');
-    res.end(JSON.stringify(data));
+app.get('/api/volumes/symbols/:id', (req, res) => {
+  PriceVolume.find({id: req.params.id}, (err, data) => {
+    if (err) {
+      return handleError(err);
+    } else {
+      res.setHeader('Content-Type', 'application/json');
+      res.end(JSON.stringify(data));
+    }
   })
 })
+//create new stock
+app.post('/api/volumes/symbols/:id', (req, res) => {
+  let newStock = new PriceVolume({ ...req.params });
+  newStock.save((err, doc) => {
+    if (err) {
+      return handleError(err);
+    } else {
+      console.log("New stock created: ", doc);
```

Comment:
is this console.log a test or do you want to keep it for the developer?
consider deleting if it was left in.

Reply:
Good call! Edited to get rid of all the console logs.